forge-gui/res/cardsfolder/upcoming/valgavoths_onslaught.txt

```diff
@@ -1,8 +1,8 @@
 Name:Valgavoth's Onslaught
 ManaCost:X X G
 Types:Instant
-A:SP$ Manifest | Dread$ True | Times$ X | RememberManifested$ True | SubAbility$ DBPutCounterAll
+A:SP$ ManifestDread | Times$ X | RememberManifested$ True | SubAbility$ DBPutCounterAll
 SVar:DBPutCounterAll:DB$ PutCounterAll | ValidCards$ Card.IsRemembered | CounterType$ P1P1 | CounterNum$ X | SubAbility$ DBCleanup

```

Comment:
Instead of `PutCounterAll | ValidCards$ Card.IsRemembered`,
just use `PutCounter | Defined$ RememberedCards`

That should be slightly faster/better? @tool4ever

Reply:
ups i was wrong, it is `Defined$ RememberedCard` without the s at the end